capture-sdk/sdk/src/main/java/net/gini/android/capture/Amount.kt

```diff
@@ -5,17 +5,36 @@ import java.text.DecimalFormat
 import java.text.DecimalFormatSymbols
 import java.util.*
 
+/**
+ * This class serves for sending the extracted amount and currency in cleanup process
+ **/
+
+/**
+ * Creates an instance of Amount class
+ *
+ * @param mBigDecimal
+ * @param mCurrency
+ *
+ */
 class Amount(
     private val mBigDecimal: BigDecimal,
     private val mCurrency: AmountCurrency
 ) {
 
+    /**
+     * For internal use only
+     *
+     * @suppress
+     */
     fun amountToPay(): String {
         val decimalFormat = DecimalFormat("0.00", DecimalFormatSymbols.getInstance(Locale.ENGLISH))
         decimalFormat.isParseBigDecimal = true
         return decimalFormat.format(mBigDecimal) + ":" + mCurrency.name
     }
 
+    /**
+     * Creates thread safe static instance of Amount class
```

Comment:
[Request] This comment should be moved to be above `@JvmStatic EMPTY`, otherwise the generated dokka will show the comment for the companion and not for `EMPTY`.